src/Sql/NotificationCenter/dbo/Stored Procedures/NotificationStatus_ReadByNotificationIdAndUserId.sql

```diff
@@ -0,0 +1,12 @@
+CREATE PROCEDURE [dbo].[NotificationStatus_ReadByNotificationIdAndUserId]
+    @NotificationId UNIQUEIDENTIFIER,
+    @UserId UNIQUEIDENTIFIER
+AS
+BEGIN
+    SET NOCOUNT ON
+
+    SELECT *
```

Comment:
I saw some `First` logic above, but can we do a `TOP 1` here? Should be impossible anyway right? What's most performant? I'd think SQL and to avoid LINQ.

Reply:
The `NotificationId` and `UserId` columns pair makes primary key for the `NotificationStatus`. It's technically not possible to get more than 1, since primary keys guarantee uniqueness and i doubt it affects the performance, since it's an exact match on PK. I have added it anyway for visibility in 006f1b4